### ISSUE TITLE
Education rewriting restart: 2020 VA education benefit rates

### ISSUE THREAD

**Issue by DanielleThierryUSDSVA** [department-of-veterans-affairs/va.gov-team#8592]
# User story

As a Veteran considering or actively using the GI Bill and other VA education benefits, I want to quickly and easily see exactly how much tuition my benefits cover for different types of education and training, and how much money I can get for additional needs like housing and books.

# Details

## Process steps to completion

**Previous steps completed before this was paused in July 2019:**

- [x] Strategy guidance
- [x] First draft in GitHub
- [X] Initial copy edit
- [X] Strategy/voice edit 

**Steps needed to complete this work now:**

- [x] Update existing Drupal page based on GitHub doc (Anne or Randi)
- [x] Update 2019 to 2020
- [ x] Add cross links to this page from other appropriate pages (Anne or Randi) http://preview-prod.vfs.va.gov/preview?nodeId=938
- [x] Ready for Strategy Review (handled in separate ticket)

(update 05/06/2020: we have moved some requirements of this story to package the pages, send to SMEs + get feedback & incorporate into other stories)

### IA details

_(note: IA and specifically urls may change based on l4 initiative)_
 - **Url:** https://www.va.gov/education/benefit-rates/
 - **Left nav label:** Benefit rates

### Content details

_(note: may need to adjust structure to work in Drupal components, as these pages were started prior to Drupal migration)_
 - **Source page:** https://www.va.gov/education/benefit-rates/
 - **SEO keywords:** post 911 gi bill rates, va education benefits rates, va benefit rates
 - **What’s needed for rewriting:**
   - Add major links/teasers for each of the benefit rates sub pages.
   - Reframe the GI Bill Comparison link as a related link.
   - Add a related link to the new [Compare Education Benefits page](https://vagov-content-pr-375.herokuapp.com/education/types-of-education-benefits/).
   - Revise intro as needed to accomodate page changes noted above.

### Github files for reference

- **Current page in GitHub:** https://github.com/department-of-veterans-affairs/vagov-content/blob/content-WIP-rewriting-education/pages/education/benefit-rates.md
 - **Branch:** https://github.com/department-of-veterans-affairs/vagov-content/tree/content-WIP-rewriting-education
 - **PR:** department-of-veterans-affairs/vagov-content#375
- **Heroku:** https://vagov-content-pr-375.herokuapp.com/

**Comment by RLHecht** [comment]
@AnneHurley This looks like our education rates landing page. I think we need to add the links to all these new rates pages on here before we can move it to the next stage. It's already a live page but linking to a legacy page for all education benefit rates.

**Comment by AnneHurley** [comment]
@RLHecht  I'll check to see what's built, and add them. Will list what's in and what's not in here.

**Comment by RLHecht** [comment]
This page now has the updated info as a draft in Drupal. Note that there is a link to the current REAP rates, but on that page it says the program was retired November 2019. I think we would need to remove the link (the REAP page is a draft in Drupal that was never published), but I don't know if we need the 2019 REAP rates page live as a historic rates page anywhere? FYSA @DanielleThierryUSDSVA @AnneHurley

**Comment by DanielleThierryUSDSVA** [comment]
Thanks, @RLHecht! Yes. I have a question for Mikki around REAP in another ticket. But I'm thinking that even if we do keep a historic rates section, we won't want to highlight it on this page. So I think we can remove from here. Thank you!

**Comment by RLHecht** [comment]
REAP rates link and teaser text was removed from this page. Ready for next steps.

**Comment by AnneHurley** [comment]
@RLHecht  I copyedited this, and I added a new link teaser to the new Compare benefits page. I added that to the top of the list. You may want to take a look and see if you think that should be in that place. Otherwise it's ready for Danielle

**Comment by DanielleThierryUSDSVA** [comment]
# Notes from reviews
(cross posted from review issue)

## 2020 VA education rates
- Updated the year in the intro text and meta description. It still had 2019, and looks like date was only updated in H1 and title tag but missed in intro text and meta description. 
- We need to fix the URL for Post-9/11 GI Bill (Chapter 33) rates. It looks like it's /2020-post-911-gi-bill-chapter-33-rates. The main rates page for the current year should be year-agnostic so that it can remain an evergreen year. I'll fix this on the actual page and then fix here to match.
- Fixed link summary for $600 Buy-Up program. It still had a reference to increasing REAP payment in there, but REAP is no longer available.
- Moved GI Bill Comparison tool up into the main section as the first link. Since this tool specifically lets the user look at and compare rates, it feels like it really should be up there in the main section rather than buried in related links. Going to check in with Mikki on this one for a gut check.

**Comment by AnneHurley** [comment]
Thanks @DanielleThierryUSDSVA for these edits. Let me know if there's anything remaining here you'd like me to fix or look at.

**Comment by lwalsh617** [comment]
@RLHecht - This page is ready for you to review. Here are some notes and questions:
![image](https://user-images.githubusercontent.com/53535663/144908117-bc2f664c-c4e9-4a0a-b124-0b9990321acd.png)
Here's the link to the preview page: 
http://preview-prod.vfs.va.gov/preview?nodeId=1007

**Comment by psanders86** [comment]
@RLHecht I know you've already given this a pass, but this page is finalized with an updated H1, meta description and page intro. Please move to validate when you are done, thank you!

http://preview-prod.vfs.va.gov/preview?nodeId=1007

**Comment by RLHecht** [comment]
Thanks, @psanders86 I did my copyedit and this is ready for the final review.

**Comment by SgardnerPMP** [comment]
Review of content in Story #36932
